src/bot.ts

```diff
@@ -109,9 +114,32 @@ export default class {
 
     const spinner = ora('📡 connecting to Discord...').start();
 
-    this.client.once('ready', () => {
+    this.client.once('ready', async () => {
       debug(generateDependencyReport());
 
+      spinner.text = '📡 Updating commands in all guilds...';
+
+      // Update commands
+      const rest = new REST({version: '9'}).setToken(this.token);
+
+      switch (this.env) {
```

Comment:
A simple if-statement would probably be cleaner here since there's only 2 cases.